pallets/offchain-worker/src/tests.rs

```diff
@@ -155,17 +155,40 @@ impl ExternalityBuilder {
 }
 
 #[test]
-fn test_chars_to_u64() {
+fn test_chars_to_u128() {
```

Comment:
As mentioned above, unit tests of helper function is better to be placed in the same file as these helper functions

Reply:
Good one. We will reorganize these functions and tests.